### PR TITLE
Use ceil for round calculations - Closes #427

### LINKED ISSUE #427: Use ceil for round calculations
Currently we use that formula for calculating round:
`Math.floor(height / 101) + (height % 101 > 0 ? 1 : 0);`
We can replace it with:
`Math.ceil(height / 101);`
That will give us exact same results when `height>=0` (always is).
Should be replaced in entire system, including SQL queries.

Issue pointed by @simonmorgenthaler

_Edit:_
For SQL queries: `CEIL(height / 101::float)`

### PR DESCRIPTION
**Pull request by 4miners**
Closes https://github.com/LiskHQ/lisk/issues/427

### CODE REVIEW

File: logic/transaction.js [hunk 1]
```diff
@@ -531,7 +526,7 @@ Transaction.prototype.apply = function (trs, block, sender, cb) {
 	this.scope.account.merge(sender.address, {
 		balance: -amount,
 		blockId: block.id,
-		round: calc(block.height)
+		round: modules.rounds.calc(block.height)
```
Comment (karmacoma) [inline logic/transaction.js]:
```
modules is not defined
```

File: logic/transaction.js [hunk 2]
```diff
@@ -542,7 +537,7 @@ Transaction.prototype.apply = function (trs, block, sender, cb) {
 				this.scope.account.merge(sender.address, {
 					balance: amount,
 					blockId: block.id,
-					round: calc(block.height)
+					round: modules.rounds.calc(block.height)
```
Comment (karmacoma) [inline logic/transaction.js]:
```
modules is not defined
```

File: logic/transaction.js [hunk 3]
```diff
@@ -560,7 +555,7 @@ Transaction.prototype.undo = function (trs, block, sender, cb) {
 	this.scope.account.merge(sender.address, {
 		balance: amount,
 		blockId: block.id,
-		round: calc(block.height)
+		round: modules.rounds.calc(block.height)
```
Comment (karmacoma) [inline logic/transaction.js]:
```
modules is not defined
```

File: logic/transaction.js [hunk 4]
```diff
@@ -571,7 +566,7 @@ Transaction.prototype.undo = function (trs, block, sender, cb) {
 				this.scope.account.merge(sender.address, {
 					balance: amount,
 					blockId: block.id,
-					round: calc(block.height)
+					round: modules.rounds.calc(block.height)
```
Comment (karmacoma) [inline logic/transaction.js]:
```
modules is not defined
```

**Comment by 4miners** [comment]
Fixed. That solution is far from perfect, but it works. ;) `library.logic.transaction/block/account` are specyfic modules, so using one of currently used approach fail here. Another options:
- require `rounds` module directly inside `logic/transaction.js`
- move `Rounds.prototype.calc` to `helpers/slots.js` or maybe create rounds helper

**Pull request merged**